user/installing-dependencies.md

```diff
@@ -329,6 +332,22 @@ addons:
 ```
 {: data-file=".travis.yml"}
 
+### Using Homebrew without addon on older macOS images
+
+It may be convenient to invoke the `brew` command directly (e.g, you write
```

Comment:
What do you think about something like

> If you're running the `brew` command directly in your build scripts, and you're using an older macOS image you'll need to update to Ruby 2.3 or newer.

And if we can tell them which is the macOS image that would be awesome. Ie macOS X or older...